components/org.wso2.carbon.identity.tenant.resource.manager/pom.xml

```diff
@@ -0,0 +1,126 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Add the licence header.

Reply:
Fixed : https://github.com/wso2-extensions/identity-governance/pull/348